app/src/components/NotificationDropdown.vue

```diff
@@ -182,31 +181,30 @@ const handleWage = async (notification: Notification) => {
   const resourceArr = getResource(notification)
 
   if (!resourceArr || resourceArr[0] !== 'wage-claim') return
-
-  updateEndPoint.value = `teams/${Number(resourceArr[1])}/cash-remuneration/claim`
-  await getWageClamAPI()
-  console.log(`wageClaim: `, wageClaim.value)
-  updateEndPoint.value = `teams/${String(route.params.id)}`
-  console.log(`endpoint`, updateEndPoint.value)
-  await getTeamAPI()
-  console.log(`cashRemunerationEip712Address: `, team.value.cashRemunerationEip712Address)
-
-  if (team.value.cashRemunerationEip712Address && wageClaim.value) {
-    const claim = {
-      employeeAddress: useUserDataStore().address,
-      hoursWorked: wageClaim.value.hoursWorked,
-      hourlyRate: wageClaim.value.hourlyRate,
-      date: Math.floor(new Date(wageClaim.value.createdAt).getTime() / 1000)
+  try {
+    updateEndPoint.value = `teams/${Number(resourceArr[1])}/cash-remuneration/claim`
+    await getWageClamAPI()
+    updateEndPoint.value = `teams/${wageClaim.value.teamId}`
+    await getTeamAPI()
+
+    if (team.value.cashRemunerationEip712Address && wageClaim.value) {
+      const claim = {
+        employeeAddress: useUserDataStore().address,
```

Comment:
Don't use this syntaxt, create the storage, do no call it like that

Reply:
Do you mean creating a variable to store `useUserDataStore().address`?

Reply:
Yes, You need to use it like this 
`const userStore = useUserDataStore()`